docs/topics/proc_deploying-the-crud-booster-using-the-oc-cli-client.adoc

```diff
@@ -102,7 +103,7 @@ NAME                             READY     STATUS      RESTARTS   AGE
 +
 Your `{value-name-app}-1-aaaaa` pod should have a status of `Running` and should be indicated as ready once it is fully deployed and started.
 
-. Once your booster is deployed and started, determine its route.
+. Once your example is deployed and started, determine its route.
```

Comment:
Once your example is deployed and started--> Once your example application is deployed and started